spec/workers/digest_email_generation_worker_spec.rb

```diff
@@ -104,7 +102,7 @@
     end
 
     context "when there are no content changes to send" do
```

Comment:
```suggestion
    context "when there are no digest items to send" do
```

This refers to messages now, so slightly misleading.